apps/vr-tests-react-components/src/stories/CalendarCompat.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../utilities/TestWrapperDecorator';
+import { Steps, StoryWright } from 'storywright';
+import { Calendar as CalendarBase, CalendarProps } from '@fluentui/react-calendar-compat';
```

Comment:
_nit:_ Split `CalendarProps` into an `import type` statement

```suggestion
import { Calendar as CalendarBase } from '@fluentui/react-calendar-compat';
import type { CalendarProps } from '@fluentui/react-calendar-compat';
```